inspirehep/modules/workflows/static/js/inspire_workflows_ui/templates/details_decision_authors.html

```diff
@@ -1,4 +1,7 @@
 <div ng-if="vm.record._extra_data._action === 'author_approval' && vm.record._workflow.status === 'HALTED'">
+<button class="btn btn-info"
+        ng-click="Utils.redirectToReviewSubmission(vm.record.id)">Review Submission
```

Comment:
It sounds like a better idea than having dedicated functions for every redirect. But where is the place to put that? (maybe somewhere in `modules/theme/static/js` ?)